parser/parser_test.go

```diff
@@ -0,0 +1,51 @@
+package parser
```

Comment:
Please add the LICENSE on the top of this file (plus a blank line before `package parser`)

```go
// Copyright 2017 Google Inc. All rights reserved.
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
// http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to writing, software distributed
// under the License is distributed on a "AS IS" BASIS, WITHOUT WARRANTIES OR
// CONDITIONS OF ANY KIND, either express or implied.
//
// See the License for the specific language governing permissions and
// limitations under the License.
```

Reply:
It looks like this was addressed, @campoy.

Reply:
Are we still waiting on this?